Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -202,6 +202,14 @@
         "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.Services/Services/Twitter",
         "CodeFile": "TwitterCode.bind",
         "Icon": "/SamplePages/Twitter Service/TwitterLogo.png"
+      },
+      {
+        "Name": "MicrosoftGraph Service",
+        "Type": "MicrosoftGraphPage",
+        "About": "The Microsoft Graph service allows you to connect to Microsoft Graph Office 365 API.",
+        "CodeUrl": "<INSERT LINK>",
```

Comment:
Don't forget to insert the link here, this can be a link that is only valid once merged.